tests/plugins/test_deepspeed_plugin.py

```diff
@@ -694,6 +695,26 @@ def test_deepspeed_multigpu_test(tmpdir, deepspeed_config):
     trainer.test(model)
 
 
+@RunIf(deepspeed=True)
+@mock.patch('platform.system', return_value='Windows')
+@mock.patch('deepspeed.init_distributed', autospec=True)
+@mock.patch.dict(os.environ, {}, clear=True)
```

Comment:
we have a conftest fixture for this already 

https://github.com/PyTorchLightning/pytorch-lightning/blob/2d682fa4618c8c70535c55c20d51e73fc367c948/tests/conftest.py#L35-L42

Reply:
Thanks, I've changed the test now to not require this